src/hdr_histogram.h

```diff
@@ -76,13 +78,17 @@ struct hdr_histogram
     int64_t sub_bucket_mask;
     int32_t sub_bucket_count;
     int32_t bucket_count;
-    atomic_int_least64_t min_value;
-    atomic_int_least64_t max_value;
     int32_t normalizing_index_offset;
-    double conversion_ratio;
     int32_t counts_len;
-    atomic_int_least64_t total_count;
     atomic_int_least64_t* counts;
+    // cache line 2
+    int64_t cache_line_padding[6]; // 48 bytes
```

Comment:
could just use uint8_t padding[48] and no need for the comment?

Reply:
could also add an offsetof assert somewhere (is there a startup routine?) which validates the expected cacheline aligned members?